spec/models/search_result_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SearchResult, type: :model do
+  describe 'validations' do
+    it { is_expected.to validate_presence_of(:keyword) }
+
+    it { is_expected.to validate_presence_of(:search_engine) }
+
+    it 'is invalid without a keyword' do
+      search_result = described_class.new(keyword: nil)
+      search_result.valid?
+      expect(search_result.errors[:keyword]).to include("can't be blank")
```

Comment:
Please add space to separate blocks:
```suggestion
      search_result = described_class.new(keyword: nil)

      search_result.valid?

      expect(search_result.errors[:keyword]).to include("can't be blank")
```